### PR TITLE
PREL-206 Contributing guide

### PR DESCRIPTION
**Pull request by nastena1606**
new file:   CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+3. Commit your changes.
+    * In the _Commit changes_ section, describe your changes.
+    * Select the **Create a new branch for this commit** and start a pull request option
+    * Click **Propose changes**.
+4. GitHub creates a branch and a commit for your changes. It loads a new page on which you can open a pull request to Percona. The page shows the base branch - the one you offer your changes for, your commit message and a diff - a visual representation of your changes against the original page. This allows you to make a last-minute review. When you are ready, click the Create pull request button.
+5. Someone from our team reviews the pull request and if everything is correct, merges it into the documentation. Then it gets published on the site.
+
+### Edit documentation locally
+
+This option is for users who prefer to work from their computer and / or have the full control over the documentation process.
```
Comment (alina-derkach-oaza) [inline CONTRIBUTING.md:31]: and/or

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+- `mkdocs.yml` - the configuration file. It includes general settings and documentation structure.
+- `docs`:
+  - `*.md` - Source markdown files.
+  - `_images` - Images, logos and favicons
+  - `css` - Styles
+  - `js` - Javascript files
+- `_resource`:
+   - `theme`:
+      - `main.html` - The layout template for hosting the documentation on Percona website
+   - overrides - The folder with the Material theme template customization for builds
```
Comment (alina-derkach-oaza) [inline CONTRIBUTING.md:132]: `overrides`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+`percona-release` documentation is written in Markdown language, so you can 
+[edit it online via GitHub](#edit-documentation-online-vi-github). If you wish to have more control over the doc process, jump to how to [edit documentation locally](#edit-documentation-locally). 
+
+Before you start, learn what [git], [MkDocs] and [Docker] are and what [Markdown] is and how to write it. For your convenience, there's also a cheat sheet to help you with the syntax. 
+
+The doc files are in the `docs` directory.
+
+### Edit documentation online via GitHub
+
+1. Click the **Edit this page** icon next to the page title. The source `.md` file of the page opens in GitHub editor in your browser. If you haven’t worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of it for you.
```
Comment (alina-derkach-oaza) [inline CONTRIBUTING.md:20]: Click the **Edit this page** icon -> Click the **pencil** icon?

File: CONTRIBUTING.md (hunk 1 above)
Comment (nastena1606) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
+# Contributing guide
+
+Welcome to Percona Software Release documentation! This documentation describes `percona-release` - the repositories management tool. `percona-release` simplifies the process of subscribing to Percona repositories which enables you to install / update Percona software and all required dependencies using the package manager of your operating system.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (nastena1606) [reply CONTRIBUTING.md]: fixed

**Pull request merged**